awx/main/tasks/facts.py

```diff
@@ -57,6 +58,29 @@ def start_fact_cache(hosts, destination, log_data, timeout=None, inventory_id=No
     return None
 
 
+def raw_update_hosts(host_list):
+    Host.objects.bulk_update(host_list, ['ansible_facts', 'ansible_facts_modified'])
+
+
+def update_hosts(host_list):
```

Comment:
is `update_hosts(host_list,max_tries=5)` not a cool thing to do here? The type in the initializer is immutable, so you won't clobber it for other callers in the fn.